fsharp-backend/tests/Tests/DvalRepr.Tests.fs

```diff
@@ -423,32 +422,38 @@ module Password =
         testNoAutoSerialization ]
 
 module LibJwt =
+
+
   let testJsonSameOnBoth =
+    let tests =
```

Comment:
I found the nesting (with extra parens and such) to be a bit confusing. Pulling the tests out like this helped me read things.